roles/rest-api/defaults/main.yaml

```diff
@@ -0,0 +1,16 @@
+---
+# Git branches
+rest_api_git_branch: develop
+
+# Paths to services folder
+rest_api_workdir: /opt/rest-api
+
+# systemd unit folder
+rest_api_service_dir: /etc/systemd/system
```

Comment:
Just hardcode this service directory. From my perspective, this will never be changed.

Reply:
ok